Update Anbox instructions for OTA-16 (#400)
ubports/docs.ubports.com: userguide/dailyuse/anbox.rst

Make sure your device is supported:

> - Meizu Pro 5

Reviewer: ```suggestion
- Meizu PRO 5
```
As per https://en.wikipedia.org/wiki/Meizu_PRO_5 and https://www.meizu.com/en/products/pro5/spec.html

- Fairphone 2
- OnePlus One
- Nexus 5

> - BQ M10 HD

Reviewer: ```suggestion
- BQ Aquaris M10 HD
```

> - BQ M10 FHD

Reviewer: ```suggestion
- BQ Aquaris M10 FHD
```

> You will need the device codename and the name of your boot partition for the installation.

Reviewer 2: this sentence can be removed

    rm anbox-boot-$CODENAME.img
    exit

.. _run-anbox-installer:

Run the Anbox installer
^^^^^^^^^^^^^^^^^^^^^^^

Once your device has the Anbox kernel installed, you can use the Anbox Tool to install the Anbox container.

#. Run ``adb shell`` from your host computer to get a shell on your Ubuntu Touch device.
#. Run the following command on your Ubuntu Touch device: ``anbox-tool install``.
#. Follow the on-screen instructions.

Now you're done! You might have to refresh the app drawer (pull down from the center of the screen and release) for the new Android apps to show up.

.. note::

>     You now have an adb server running inside your phone. This guide asks you to run some ``adb`` commands, sometimes on your computer, other times on your phone. Carefully check on which device you are!

Reviewer: ```suggestion
    You now have an ADB server running on your Ubuntu Touch device. This guide asks you to run some ``adb`` commands, sometimes on your computer, other times on the device itself. Carefully check which device you are on!
```

> You can check that adb server is correctly running locally on your phone by opening the terminal app and enter ``adb devices``. You should see something like::

Reviewer: ```suggestion
You can check that ADB server is correctly running locally on your phone by opening the terminal app and enter ``adb devices``. You should see something like::
```

Troubleshooting
---------------

> - When you want to install an apk but get the error ``Invalid APK file`` that error could also mean "file not found"

Reviewer: ```suggestion
- When you want to install an APK, but get the error ``Invalid APK file`` that error could also mean "file not found"
```

>   - Check that you typed the file name correctly

Reviewer: ```suggestion
  - Check that you typed the filename correctly
```
